core/src/main/java/org/ehcache/Ehcache.java

```diff
@@ -83,9 +83,9 @@
 /**
  * @author Alex Snaps
  */
-public class Ehcache<K, V> implements Cache<K, V>, UserManagedCache<K, V>, PersistentUserManagedCache<K, V> {
+public class Ehcache<K, V> implements Cache<K, V>, UserManagedCache<K, V> {
 
-  private final StatusTransitioner statusTransitioner;
+  protected final StatusTransitioner statusTransitioner;
```

Comment:
Not really happy with having to do that, however I have not found another solution and really want to have user managed persistent related code in a subclass.